### PR TITLE
Feature/blink led

### LINKED ISSUE #4: Feature blink-led
First feature for input device

### PR DESCRIPTION
**Pull request by rarriens**
Closes #4 
Implemented a feature to make a Led blink with a certain period.
Output pin needs to be specified for the respective board(LOLIN D32 pro v2).

### CODE REVIEW

File: include/Led.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+#include <Arduino.h>
+
+class Led {
+
+    public:
+        Led(int pin);
+        void Blink(int led_delay);
+    private:
+    int output_pin; // Each Led instance has a output pin
```
Comment (martijnvandermarel) [inline include/Led.h]: tab for readability?

File: platformio.ini [hunk 2]
```diff
@@ -12,3 +12,4 @@
 platform = espressif32
 board = lolin_d32_pro
 framework = arduino
+upload_port = COM[7]
```
Comment (martijnvandermarel) [inline platformio.ini]: Perhaps comment that this depends per user/pc?

File: src/main.cpp [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+#include "Led.h"
 
-void setup() {
-  // put your setup code here, to run once:
+// Define output pin and period of blink
+#define Pinout 1
+#define led_delay 50 // This value x2 is the period in ms
+
+// Create Led instance
+Led led(Pinout);
```
Comment (martijnvandermarel) [inline src/main.cpp:9]: is it possible to move this to setup()?

File: src/main.cpp [hunk 4]
```diff
@@ -1,9 +1,18 @@
 #include <Arduino.h>
+#include "Led.h"
 
-void setup() {
-  // put your setup code here, to run once:
+// Define output pin and period of blink
+#define Pinout 1
+#define led_delay 50 // This value x2 is the period in ms
```
Comment (martijnvandermarel) [inline src/main.cpp:6]: If possible, make these 'normal' variables and declare them in setup()?

File: platformio.ini (hunk 2 above)
Comment (rarriens) [reply platformio.ini]: Commented out line, added comment.
Platformio apparently detects this automatically

File: src/main.cpp (hunk 3 above)
Comment (rarriens) [reply src/main.cpp:9]: No this is not possible. This line creates an instance of the Led class, which you cannot do within the setup().

File: src/main.cpp (hunk 4 above)
Comment (rarriens) [reply src/main.cpp:6]: The setup() function is used to initialize variables and define pin mode. When you define them as normal variables, they won't be in the scope of loop()

File: include/Led.h (hunk 1 above)
Comment (rarriens) [reply include/Led.h]: Fixed!

**Review by martijnvandermarel — APPROVED** [review]

**Pull request merged**